src/components/utils/caseUtils.ts

```diff
@@ -11,21 +11,30 @@ import {
     getCaseOriginalName,
 } from '../../utils/rest-api';
 import { useIntl } from 'react-intl';
-import { ElementType, useSnackMessage } from '@gridsuite/commons-ui';
+import {
+    ElementAttributes,
+    ElementType,
+    useSnackMessage,
+} from '@gridsuite/commons-ui';
 import { useCallback, useState } from 'react';
 
-const downloadCases = async (uuids: string[]) => {
-    for (const uuid of uuids) {
-        const result = await downloadCase(uuid);
-        let name = await getCaseOriginalName(uuid);
+const downloadCases = async (selectedCases: ElementAttributes[]) => {
+    for (const selectedCase of selectedCases) {
+        const result = await downloadCase(selectedCase.elementUuid);
```

Comment:
the fetch `downloadCase` need to get also the format so as not to call the `getCaseOriginalName`
In an another US